company/how-to-guides-for-staff/how-to-update-handbook.md

```diff
@@ -78,6 +78,10 @@ All Handbook pages are written in Markdown, which is also the language used to p
 
 You can update the left-hand navigation in the [SUMMARY.md](https://github.com/mattermost/mattermost-handbook/blob/0.2.1/SUMMARY.md) file.
 
+**Important note:**
+
+Gitbook dynamically changes the URL based on the location in the TOC. This means that when things are moved around, the previous link is broken. There is a redirect file that we use to prevent this in the `gitbook.yaml` file. Please tag @jason.blais or @justine.geffen for assistance with this if needed.
```

Comment:
```suggestion
GitBook dynamically changes the URL based on the location in the table of contents. This means that when a page changes its location, the previous link results in a 'page not found' error.

There is a redirect file that we use to prevent this in the `gitbook.yaml` file. Please mention @jason.blais or @justine.geffen in Mattermost (@jasonblais or @justinegeffen in GitHub) for assistance if needed.
```